evolver/base.py

```diff
@@ -94,6 +98,22 @@ def shallow_model_dump(self):
         return dict(self)
 
 
+CreatedTimestampField = partial(
+    pydantic.Field, description="The creation timestamp", default_factory=datetime.datetime.now
+)
+ExpireField = partial(
+    pydantic.Field,
+    default=None,
+    description="The amount of time after which the associated object is considered stale. "
+    "`None` := forever (the default).",
+)
```

Comment:
I feel like there should be a more accepted way of doing this, i.e., using ``Annotated``. However, I tried this but couldn't get ``pydantic.Field`` param ``default_factory`` to work.